content/chainguard/chainguard-images/working-with-images/security-advisories/managing-advisories/index.md

```diff
@@ -0,0 +1,193 @@
+
+In this guide, you will walk through using `wolfictl` to create an advisory for a vulnerable package. You’ll also learn how to update this advisory as more information about the vulnerability is disclosed over time. To follow along, you will need to have [`git`](https://git-scm.com/) and the [Go programming language](https://go.dev/dl/) installed on your machine. 
+
+This guide will focus on the packages and advisories issued for Wolfi.
+
+
+## How to Install `wolfictl`
+
+To work with security advisories, you will need to install the `wolfictl` tool onto your machine. First, execute the following command in your terminal to clone the `wolfictl` repository locally, and navigate to it. 
+
+```sh
+git clone git@github.com:wolfi-dev/wolfictl.git wolfictl && cd $_
+```
+
+Then, using `go`, install the `wolfictl` tool:
+
+```sh
+go install
+```
+If you encounter any errors during installation, your installed version of `go` may be out of date. You can check what version of `go` you have installed by running `go version` in your terminal. Check the [`go.mod` file](https://github.com/wolfi-dev/wolfictl/blob/main/go.mod) in the `wolfictl` repository to determine what version of `go` you will need, and be sure to update `go` to this version or a later release to continue. Alternatively, you may need to add the `wolfictl` binary to your `$PATH` after installation if your system does not recognize the command.
+
+You can verify that you have successfully installed `wolfictl` by executing the `wolfictl version` command in your terminal.
+
+```sh
+wolfictl version
+```
+A successful installation of `wolfictl` will display output similar to the following. Note that the exact output will vary over time as new `wolfictl` versions are released. 
+
+```Output
+ __        __   ___    _       _____   ___    ____   _____   _
+ \ \      / /  / _ \  | |     |  ___| |_ _|  / ___| |_   _| | |
+  \ \ /\ / /  | | | | | |     | |_     | |  | |       | |   | |
+   \ V  V /   | |_| | | |___  |  _|    | |  | |___    | |   | |___
+    \_/\_/     \___/  |_____| |_|     |___|  \____|   |_|   |_____|
+wolfictl: A CLI helper for developing Wolfi
+
+GitVersion:    devel
+GitCommit:     6c98dc69a559192575d085d87fd916d8281dd67d
+GitTreeState:  clean
+BuildDate:     2024-07-23T01:57:17
+GoVersion:     go1.22.5
+Compiler:      gc
+Platform:      darwin/arm64
+```
+
+In the next section, you will complete your local setup so you can begin using the `wolfictl` tool to work with advisories.
+
+## Cloning Package and Advisory Repositories
+
+Before you can interact with security advisories, the  `wolfictl` tool will need access to existing Wolfi package and advisory information.
```

Comment:
My issue here was more that it's funky to have two code blocks back to back without a sentence or something between them. Readers want to know why they're running a command and it's not intuitive to have to look back for the context.

Personally, I try to avoid having two different commands in the same block, but in this case (where they're two commands that do basically the same thing) it's not that big of a deal.